### PR TITLE
fix(statics): update avaxp network to use number instead of bigint

### PR DESCRIPTION
**Pull request by jicli**
bigint cannot be parsed when download transaction
Ticket: STLX-17918

### CODE REVIEW

File: modules/statics/src/networks.ts [hunk 1]
```diff
@@ -169,19 +169,19 @@ class AvalancheP extends Mainnet implements AvalancheNetwork {
...
+  createSubnetTx = 1000000000; // 1 AVAX
+  createChainTx = 1000000000; // 1 AVAX
+  creationTxFee = 10000000; // 1 CENTIAVAX
   minConsumption = 0.1;
   maxConsumption = 0.12;
-  maxStakingDuration = BigInt(31536000); // 1 year
-  maxSupply = BigInt(720000000) * BigInt(1000000000); // 720 mil tokens
-  minStake = BigInt(1000000000) * BigInt(2000);
+  maxStakingDuration = 31536000; // 1 year
+  maxSupply = 720000000 * 1000000000; // 720 mil tokens
```
Comment (joey-bai) [inline modules/statics/src/networks.ts:179]: this exceeds JS number overflow limit and will likely cause you problems if you try to do math with it.
this is `7.2 * 10^17`
safe overflow limit for JS is ~ `9 * 10^15`

Comment (joey-bai) [reply modules/statics/src/networks.ts:179]: alternative is to make this a string instead, and call it `maxSupplyString`

Comment (joey-bai) [reply modules/statics/src/networks.ts:179]: when you consume it, you can use any BigNumber/BigInt library to safely do math on it.

Comment (jicli) [reply modules/statics/src/networks.ts]: updated it to string

**Review by joey-bai — APPROVED** [review]
LGTM - will defer to SDK team on usage of string instead of bigint

**Review by tylerlevine — CHANGES_REQUESTED** [review]
nit: do not include the type in the name, that information is already in the type

Commit does need to identify this as a breaking change though, since you are making a change that could break calling code.

**Review by tylerlevine — APPROVED** [review]

**Pull request merged**